check_stability.py

```diff
@@ -1,6 +1,7 @@
 from __future__ import print_function
 
 import argparse
+from ConfigParser import SafeConfigParser
```

Comment:
Why did this move? Convention is that `from…` imports go after plain `import` and that sorting is by unicode codepoint.